frontend/src/metabase/reference/databases/DatabaseDetailContainer.jsx

```diff
@@ -0,0 +1,76 @@
+/* eslint "react/prop-types": "warn" */
+import React, { Component } from 'react';
+import PropTypes from "prop-types";
+import { connect } from 'react-redux';
+
+import DatabaseSidebar from './DatabaseSidebar.jsx';
+import SidebarLayout from 'metabase/components/SidebarLayout.jsx';
+import DatabaseDetail from "metabase/reference/databases/DatabaseDetail.jsx"
+
+import * as metadataActions from 'metabase/redux/metadata';
+import * as actions from 'metabase/reference/reference';
+
+import {
+    getDatabase,
+    getDatabaseId,
+    getIsEditing
+} from '../selectors';
+
+
+const mapStateToProps = (state, props) => ({
+    database: getDatabase(state, props),
+    databaseId: getDatabaseId(state, props),
+    isEditing: getIsEditing(state, props)
+});
+
+const mapDispatchToProps = {
+    ...metadataActions,
+    ...actions
+};
+
+@connect(mapStateToProps, mapDispatchToProps)
+export default class DatabaseDetailContainer extends Component {
+    static propTypes = {
+        params: PropTypes.object.isRequired,
+        location: PropTypes.object.isRequired,
+        database: PropTypes.object.isRequired,
+        isEditing: PropTypes.bool
+    };
+
+    async fetchContainerData(){
+        await actions.rFetchDatabaseMetadata(this.props, this.props.databaseId);
```

Comment:
I second @attekei's comment. It's not obvious that `r` stands for `wrapped`. 

I think any time you have to explain what the meaning of the hungarian notation prefixes you're putting in front of your variables you should stop and consider whether introducing them is a good idea. A more verbose name like `wrappedFetchDatabaseMetadata` at least isn't going to have a bunch of potential contributors and future new hires scratching their head or putting `r` in from of random variables because they don't understand the pattern